### PR TITLE
Docs for `FROM DOCKERFILE -f`

### PR DESCRIPTION
**Pull request by vladaionescu**
Re https://github.com/earthly/earthly/issues/619

### CODE REVIEW

File: docs/earthfile/earthfile.md [hunk 1]
```diff
@@ -751,6 +751,10 @@ This feature is currently in **Beta** and it has the following limitations:
 
 #### Options
 
+##### `-f <dockerfile-path>`
+
+Specify an alternative Dockerfile to use. The `<dockerfile-path>` can be either a path on the host system, relative to the current Earthfile, or an artifact reference pointing to the `Dockerfile`.
```
Comment (dchw) [inline docs/earthfile/earthfile.md:756]:
```suggestion
Specify an alternative Dockerfile to use. The `<dockerfile-path>` can be either a path on the host system, relative to the current Earthfile, or an artifact reference pointing to a `Dockerfile`.
```

Comment (dchw) [reply docs/earthfile/earthfile.md:756]: So a Dockerfile as an artifact must be named +some-target/Dockerfile? At least, thats how I initially parsed this sentence. Assuming that i s not true, I would probably phrase it like this

Comment (dchw) [reply docs/earthfile/earthfile.md:756]: Side note: I went to add a link to what an "artifact reference" is. The closest I found is the description in `COPY`. This seems like a fundamental term I should find a better definition for when I search "artifact reference" over our docs.

**Review by dchw — APPROVED** [review]
Small nit if I understood right

File: docs/earthfile/earthfile.md (hunk 1 above)
Comment (vladaionescu) [reply docs/earthfile/earthfile.md:756]: It doesn't have to be named `Dockerfile` - good catch.

Ah right.. it's in `target-ref.md`. I'll add a link to that too.

**Pull request merged**